src/main/Shopware/Connect/Struct/ShopConfiguration.php

```diff
@@ -44,11 +44,21 @@ class ShopConfiguration extends Struct
 
     /**
      * Defines how the shipping costs are calculated for merchant and consumer.
+     * This field shows to supplier how shipping costs should be calculated.
      *
      * @var string
      */
     public $shippingCostType = 'remote';
 
+    /**
+     * Defines how the shipping costs are calculated when an account
+     * makes orders with products from current supplier.
+     * This field shows to merchant how shipping costs will be calculated.
+     *
+     * @var string
+     */
+    public $merchantShippingCostType = 'remote';
```

Comment:
maybe we need to find a better name here ;) i think its quite confusing
and am I right that it is possible if i have supplier 1 in shop configuration to merchant 1 `$merchantShippingCostType = 'remote'` but in shopconfiguration to merchant 2 it is `$merchantShippingCostType = 'fixed'`
or isn't this a value that is set for each offer?

Reply:
yeah the name is confusing. ```shippingCostType``` property is used to show you how you need to calculate shipping costs as supplier. 
So if you have shopA with shopId 44 and this shop is supplier,
shopB with shopId 68 and this shop is merchant.

Let's say that shopB has 5 remote products from shopA. Their shipping cost types are defined in account_contact table in SN. What will be replicated to the shops?
ShopA will receive ShopConfiguration struct
id: 68
name: ShopB
shippingCostType: remote
merchantShippingCostType: all
sharedKey: some long string here
================================================
ShopB will receive ShopConfiguration struct
id: 44
name: ShopB
shippingCostType: all
merchantShippingCostType: remote
sharedKey: some long string here

In case where ShopB is supplier and ShopA makes orders, it will use shippingCostType: all,
but when ShopB makes an order with products from ShopA "merchantShippingCostType: remote" will be used in ShopB. This is needed, because when remote shopping is enabled checkout has different behavior.

Let's summarize: Until now only shippingCostType: remote was available, but this field is used only when you're supplier. If you are merchant there is no way to access which shipping cost type is used by supplier to calculate shipping costs.

Reply:
maybe we can change `shippingCostType` to `fromShopShippingCostType` because it defines what the type is what the shipping type is if i provide products for that shop and then
change `merchantShippingCostType` to `toShopSchippingCostType` because thats the shipping type if i receive products 
maybe i accidently mixed up `shippingCostType` and `merchantShippingCostType`

Reply:
or maybe instead of toShop and FromShop we can say
`importedProductsShippingCostType`and `exportedProductsShippingCostType`
what do you think?